### PR TITLE
Add JavaString to the vtl value mapper

### PR DESCRIPTION
**Pull request by adamclerk**
My second attempt at adding JavaString to the value mapper.

Please provide feedback. 

`schemaTest.test.js` was throwing an error without the additional quotes in `update-request.txt`. I'm not sure if the bug is in new valueMapper or in the mapping template.

I also added some additional tests around the value mapper.

### CODE REVIEW

File: packages/appsync-emulator-serverless/vtl.js [hunk 1]
```diff
@@ -37,6 +47,25 @@ const toJSON = value => {
   return value;
 };
 
+class JavaString {
+  constructor(str) {
+    this.strVal = str;
+  }
+
+  replaceAll(...args) {
```
Comment (cbaron) [inline packages/appsync-emulator-serverless/vtl.js]: nitpick: do you mind naming the arguments here?

Comment (adamclerk) [reply packages/appsync-emulator-serverless/vtl.js]: done

**Comment by lightsofapollo** [comment]
Approach LGTM here so far! Thanks for your contributions :)

**Comment by adamclerk** [comment]
@lightsofapollo  @cbaron I'll leave it to you to merge.

**Comment by cbaron** [comment]
Currently, it is done manually :) -- I would be open to changing that.

Just released: 

```+ @conduitvc/appsync-emulator-serverless@0.12.3```

**Pull request merged**